src/SFA.DAS.EAS.Portal/Database/Models/AccountLegalEntity.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Models;
+
+namespace SFA.DAS.EAS.Portal.Database.Models
+{
+    public class AccountLegalEntity : IAccountLegalEntityDto
```

Comment:
Why all the interfaces for the read model?  It's not going to be polymorphic.

Reply:
The interfaces are to tie together the write model and read DTOs, and make sure they don't deviate.

Reply:
Out of date now, as the read and write model are the same.